modules/rhcos-install-iscsi-ibft.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-install-iscsi-ibft_{context}"]
+= Installing {op-system} on an iSCSI boot device using iBFT
+
+On a completely diskless machine, the iSCSI target and initiator values can be passed through iBFT. iSCSI multipathing is also supported.
+
+.Prerequisites
+. You are in the {op-system} live environment.
+. You have an iSCSI target you want to install {op-system} on.
+. Optional: you have multipathed your iSCSI target.
+
+.Procedure
+
+. Mount the iSCSI target from the live environment:
++
+[source,text]
+----
+$ iscsiadm \
+    --mode discovery \
+    --type sendtargets
+    --portal 192.168.0.10 \ <1>
+    --login
+----
+<1> The IP address of the target portal.
+
+. Optional: enable multipathing and start the daemon:
++
+[source,text]
+----
+$ mpathconf --enable && systemctl start multipathd.service
+----
+
+. Install {op-system} onto the iSCSI target using the necessary kernel arguments, specifying `ip=ibft`:
++
+[source,text]
+----
+$ coreos-installer install \
+    /dev/disk/by-path/<IP_address>:<port>-iscsi-<target_iqn>-lun-<lun> \
+    --append-karg rd.iscsi.firmware=1 \ <1>
+    --console ttyS0 \
+    --ignition-file <path_to_file>
+----
+<1> The iSCSI parameter is read from the BIOS firmware.
```

Comment:
This is missing the mention that `rd.multipath=default` is needed if multipathing is used.